metrics/run_all_metrics.sh

```diff
@@ -17,6 +17,14 @@
 set -e
 
 SCRIPT_PATH=$(dirname "$(readlink -f "$0")")
+source "${SCRIPT_PATH}/lib/common.bash"
+
+# Check arguments
+if [[ ! -v RUNTIME ]]; then
+    die Variable RUNTIME must be set to the path of your CC runtime binary
```

Comment:
Sorry to be a bit of a pedant :-) - it is not 'to the path' it is 'to the name'. Specifically, it is the name of the runtime as configured into Docker. For instance, for CC2.x the name is `cor`, but the actual binary is `cc-oci-runtime`. We use the name here as that is what we pass to the docker run commands with `--runtime=xxx`.